src/components/Dashboard/TodoList.vue

```diff
@@ -774,6 +790,19 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     return false
   }
 
+  /** Whether to show the Delete button only for a draft item. */
+  showDeleteOnly (item: TodoItemIF): boolean {
+    switch (item.name) {
+      case FilingTypes.ALTERATION:
+      case FilingTypes.DISSOLUTION:
+      case FilingTypes.SPECIAL_RESOLUTION:
+        // when NIGS, non-staff can only delete item (staff can resume, etc)
+        return (!this.isGoodStanding && !this.isRoleStaff)
+      default:
+        return false
+    }
+  }
```

Comment:
I previously had a new property on the item for this but I think this is better: it identifies the affected filings in one place (and should be easier to refactor TodoList like I did with FilingHistoryList previously).